### PR TITLE
fixtures to add admin users

### PR DESCRIPTION
**Pull request by si202cam**

### CODE REVIEW

**Comment by si202cam** [comment]
issue #33

File: mws/mwsauth/fixtures/mwsuser_admin_users.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- model: mwsauth.MWSUser
+  fields: { uid: 178460, user: amc203 }
+- model: mwsauth.MWSUser
+  fields: { uid: 53956, user: rjw53 }
```
Comment (ph448) [inline mws/mwsauth/fixtures/mwsuser_admin_users.yaml]: this user should be rjw57 with the associated uid

Comment (si202cam) [reply mws/mwsauth/fixtures/mwsuser_admin_users.yaml]: Thanks Peter now corrected.

**Comment by ph448** [comment]
fixes uisautomation¡#33

**Pull request merged**